nomic/project.py

```diff
@@ -163,99 +175,81 @@ def _get_index_job(self, job_id: str):
 
         return response.json()
 
-    def _validate_and_correct_user_supplied_metadata(
-        self, data: List[Dict], project, replace_empty_string_values_with_string_null=True
-    ):
+    def _validate_and_correct_arrow_upload(
+        self, data: pa.Table, project: "AtlasProject"
```

Comment:
We need to test that empty strings to do break things or cause strange behavior on search